client/my-sites/marketplace/pages/marketplace-product-install/index.tsx

```diff
@@ -492,8 +492,14 @@ const MarketplaceProductInstall = ( {
 			<QueryActiveTheme siteId={ siteId } />
 			{ siteId && <QueryJetpackPlugins siteIds={ [ siteId ] } /> }
 			<Masterbar className="marketplace-plugin-install__masterbar">
-				<WordPressWordmark className="marketplace-plugin-install__wpcom-wordmark" />
-				<Item>{ translate( 'Marketplace installation' ) }</Item>
+				<Global
+					styles={ css`
+						body {
+							--masterbar-height: 72px;
+						}
+					` }
+				/>
```

Comment:
Can we not have this on a style sheet instead?

Reply:
I don't think so, `Global` allows us to override the global masterbar height